pyproject.toml

```
@@ -64,8 +53,48 @@ omit = [
   "src/vendor/*"
 ]
 
-[tool.coverage.report]
-fail_under = 86
+
+[tool.isort]
+profile = "black"
+line_length = 80
+multi_line_output = 3
+skip = ["./src/vendor"]
+
+
+[tool.mypy]
+check_untyped_defs = true
+disallow_untyped_defs = true
+no_implicit_optional = true
+warn_redundant_casts = true
+plugins = "sqlalchemy.ext.mypy.plugin"
+
+[[tool.mypy.overrides]]
+module = "mypy-tests.*"
+disallow_untyped_defs = false
+
+[[tool.mypy.overrides]]
+module = "mypy-vendor.*"
```

```suggestion
module = "vendor.*"
```
this should solve it I think

---

Yeah.  I knew somehting was a bit fishy with that one. 😉